IEEE Xplore.js

```diff
@@ -27,46 +27,28 @@ function detectWeb(doc, url) {
 	return false;
 }
 
-function doWeb(doc, url) {
-	var hostRe = new RegExp("^(https?://[^/]+)/");
-	var hostMatch = hostRe.exec(url);
-
-	var articleRe = /[?&]ar(?:N|n)umber=([0-9]+)/;
-	var m = articleRe.exec(url);
+function getSearchResults(doc, checkOnly) {
```

Comment:
use this in `detectWeb`

Also, feel free to tidy up the target regexp while we're at this. `^https?://([^/]+\\.)?ieeexplore\\.ieee\\.org/([^#]+[&?]arnumber=\d+|search/(searchresult|selected).jsp|xpl\\/(mostRecentIssue|tocresult).jsp\\?)`